docs/fastdds/dds_layer/core/waitsets/waitsets.rst

```diff
@@ -11,20 +11,21 @@ to notify communication status changes (including arrival of data) to the applic
 
 This mechanism is wait-based. Its general use pattern is as follows:
 
-* The application indicates which relevant information it wants to get, by means of Condition
-  objects (GuardCondition, StatusCondition, or ReadCondition) and attaching them to a wait-set.
-* It then waits on that wait-set until the trigger value of one or several Condition objects become
-  true.
-* It then uses the result of the wait (i.e., the list of Condition objects with
+* The application indicates which relevant information it wants to get, by means of :ref:`api_pim_condition`
+  objects (:ref:`api_pim_guardcondition`, :ref:`api_pim_statuscondition`, or ReadCondition)
+  and attaching them to a :ref:`api_pim_waitset`.
+* It then waits on that :ref:`api_pim_waitset` until the trigger value of one or several :ref:`api_pim_condition`
+  objects become true.
+* It then uses the result of the wait (i.e., the list of :ref:`api_pim_condition` objects with
   trigger_value == true) to actually get the information by calling:
 
-  * get_status_changes and then get_<communication_status> on the relevant Entity, when the
+  * |StatusCondition::get_status_changed-api| and then get_<communication_status> on the relevant Entity, when the
     condition is a StatusCondition and the status changes refer to plain communication status.
-  * get_status_changes and then get_datareaders on the relevant Subscriber, when the condition is a
+  * |StatusCondition::get_status_changed-api| and then get_datareaders on the relevant Subscriber, when the condition is a
     StatusCondition and the status changes refer to DataOnReaders.
-  * get_status_changes and then read/take on the relevant DataReader, when the condition is a
+  * |StatusCondition::get_status_changed-api| and then read/take on the relevant DataReader, when the condition is a
```

Comment:
I suggest linking read/take to the API reference `DataReader::read`/`DataReader::take`.